main.go

```diff
@@ -93,6 +93,11 @@ func main() {
 	}
 
 	// Migrate the schema
+	err = migration.Migrate(db)
+	if err != nil {
+		panic(err.Error())
```

Comment:
Are we sure we don't need to require an exclusive access lock to the version table here?

```
LOCK TABLE version IN ACCESS EXCLUSIVE MODE;
```

As far as I can see in the docs ACCESS EXCLUSIVE is the only lock mode that will cause a SELECT to not be possible and has to wait for the other tx to finish. Without it, won't 2 starting cores just see the same version and attempt to apply the same version, one failing and the other work due to update/insert/create related locks?

Lock doc: https://www.postgresql.org/docs/9.5/static/explicit-locking.html
http://big-elephants.com/2013-09/exploring-query-locks-in-postgres/